starfish/image/_filter/linear_unmixing.py

```diff
@@ -12,7 +12,7 @@
 
 class LinearUnmixing(FilterAlgorithmBase):
 
-    def __init__(self, coeff_mat: np.ndarray, clip_method: int=1) -> None:
+    def __init__(self, coeff_mat: np.ndarray) -> None:
```

Comment:
was clip_method supposed to be deleted?

Reply:
It's unused.

Reply:
@kevinyamauchi I think you added this.  Let me know if it's a problem.